bodhi/server/renderers.py

```diff
@@ -108,6 +108,15 @@ def link_dict(obj):
                 # on the same line
                 getattr(feed_item, name)(getter(value))
 
+        """
+        This part of the code will handle cases when FeedEntry object doesn't have a title
+        and set its hyperlink as the title to prevent exception being thrown.
+        """
+        for key, feed_entry in enumerate(feed._FeedGenerator__feed_entries):
+            if not feed_entry._FeedEntry__rss_title:
+                feed._FeedGenerator__feed_entries[key]._FeedEntry__rss_title = \
```

Comment:
This code uses undocumented internal APIs from feed generator, which is problematic because they can change their API without notice. We should instead alter ```getters['comments']['title']``` above to do this for us.

Reply:
Instead of using the comment's link as the title, let's use its id.

Reply:
You could make the title be ```"{} comment #{}".format(comment.update.alias, comment.id)```.